### DISCUSSION TITLE
Guidance

### DISCUSSION THREAD

**Discussion by carlynlee** [NASA-AMMOS/slim#153] [General]
I am working on some scripts to help update a public facing website [ipnpr.jpl.nasa.gov](http://ipnpr.jpl.nasa.gov/) with new articles. Really I am one of two users of the scripts, but that might change in the future. How can I use SLIM?

**Comment by riverma** [comment]
Hi @carlynlee - thanks for reaching out! 

At a basic level, if your scripts are expected to be reused and are not a one-time thing, we'd recommend setting up a repository to house the scripts (as opposed to using something like [GitHub Gists](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists)). You also never know what code may evolve to - so sometimes it's helpful to create the groundwork for growth should it occur. 

If you go the repository route, we'd recommend you review the [SLIM Getting Started Checklist](https://nasa-ammos.github.io/slim/docs/guides/checklist#checklist) for recommend (in order of priority) things your repository should include. In fact, try using our [repository starter kit](https://nasa-ammos.github.io/slim/docs/guides/checklist#repository-starter-kit) to bootstrap your project with the needed best practices automatically!
